staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1360,8 +1360,14 @@ message AttachResponse {
 message PortForwardRequest {
     // ID of the container to which to forward the port.
     string pod_sandbox_id = 1;
+    // Name of pod
+    string pod_name = 2;
+    // Namespace of pod
+    string pod_namespace = 3;
+    // Uid of pod
+    string pod_uid = 4;
     // Port to forward.
-    repeated int32 port = 2;
```

Comment:
I don't think you can do this, changing the proto tags breaks compatibility.

Reply:
Thanks for the review, CI test failures may be caused by this. Now I'll update it and see if the CI works.